process/block/metablock_test.go

```diff
@@ -1254,17 +1253,22 @@ func TestMetaProcessor_CreateShardInfoShouldWorkNoHdrAddedNotFinal(t *testing.T)
 
 	prevRandSeed := []byte("prevrand")
 
-	lastNodesHdrs := mp.LastNotarizedHdrs()
+	lastNotarizedHdrs := mp.LastNotarizedHdrs()
 	for i := uint32(0); i < shardNr; i++ {
```

Comment:
extrac to a separate function. as it is used in the next tests, a few more times.

Reply:
Done